build.sh

```diff
@@ -75,19 +74,21 @@ while getopts "p:t:c:m:hvdfa" opt; do
 done
 
 case ${BUILD_TARGET} in
-  hera | orion)
-    echo "Building RDASApp on $BUILD_TARGET"
+  hera | orion | hercules)
+    echo "Building GDASApp on $BUILD_TARGET"
+    export PS1="[\u@\h \W]\$ " #clt to avoid unbound PS1 error
```

Comment:
This line was added for building on herea, because, otherwise, the building process would abort complain unbound PSI. 
Seems conda was invoked somewhere.  I will see if a better fix could be found